torch/csrc/jit/passes/quantization.cpp

```diff
@@ -2485,7 +2485,15 @@ void SwapDeQuant(std::shared_ptr<Graph>& graph) {
   }
 }
 
-void QuantFusion(std::shared_ptr<Graph>& graph) {
+void QuantFusion(std::shared_ptr<Graph>& graph, bool is_dynamic) {
+  if (is_dynamic) {
+    for (const auto& item : dynamic_quant_fusion_pattern_and_replacements()) {
+      SubgraphRewriter rewriter;
+      rewriter.RegisterRewritePattern(item.first, item.second);
+      rewriter.runOnGraph(graph);
+    }
+    return;
+  }
```

Comment:
could you use the same code?
Also I think you'll need to rebase, return type for quant_fusion_pattern_and_replacements is changed